Tools/clinic/clinic.py

```diff
@@ -2550,6 +2554,8 @@ def __init__(cls, name, bases, classdict):
         add_c_converter(cls)
         add_default_legacy_c_converter(cls)
 
+from typing import Type
```

Comment:
Nit: should move this to the top with the other imports

Reply:
Yeah, just laziness since I was testing out stuff :)